docs/source-1.0/spec/core/selectors.rst

```diff
@@ -1608,43 +1608,43 @@ Selectors are defined by the following ABNF_ grammar.
                                          :/ `selector_forward_recursive_neighbor`
                                          :/ `selector_variable_set`
                                          :/ `selector_variable_get`
-    selector_shape_types                 :"*" / `smithy:identifier`
+    selector_shape_types                 :"*" / `smithy:Identifier`
     selector_forward_undirected_neighbor :">"
     selector_reverse_undirected_neighbor :"<"
     selector_forward_directed_neighbor   :"-[" `selector_directed_relationships` "]->"
     selector_reverse_directed_neighbor   :"<-[" selector_directed_relationships "]-"
-    selector_directed_relationships      :`smithy:identifier` *("," `smithy:identifier`)
+    selector_directed_relationships      :`smithy:Identifier` *("," `smithy:Identifier`)
     selector_forward_recursive_neighbor  :"~>"
     selector_attr                        :"[" `selector_key` [selector_attr_comparison] "]"
     selector_attr_comparison             :`selector_comparator` `selector_attr_values` ["i"]
-    selector_key                         :`smithy:identifier` ["|" `selector_path`]
+    selector_key                         :`smithy:Identifier` ["|" `selector_path`]
     selector_path                        :`selector_path_segment` *("|" `selector_path_segment`)
     selector_path_segment                :`selector_value` / `selector_function_property`
-    selector_value                       :`selector_text` / `smithy:number` / `smithy:root_shape_id`
-    selector_function_property           :"(" `smithy:identifier` ")"
+    selector_value                       :`selector_text` / `smithy:number` / `smithy:RootShapeId`
```

Comment:
Was the intent to re-write the casing everywhere like the source-2.0/spec? Or only partially? I think we should be consistent, one way or the other.

Reply:
The goal is to re-write the casing everywhere like the source-2.0/spec, but not sure if we wanted to include it all in 1 PR.

Reply:
I'm OK with including it here. I'd rather have them consistent.

Reply:
Moved ABNF rewrite to #1357.